src/modules/components/proposal/proposalActions/actions/generators/proposalActionTokenMint.ts

```diff
@@ -0,0 +1,41 @@
+import { ProposalActionType } from '../../proposalActionsTypes';
+import { type IProposalActionTokenMint } from '../../proposalActionsTypes/proposalActionTokenMint';
+
+export const generateProposalActionTokenMint = (
+    action?: Partial<IProposalActionTokenMint>,
+): IProposalActionTokenMint => ({
+    type: ProposalActionType.TOKEN_MINT,
+    from: '0x742d35Cc6634C0532925a3b844Bc454e4438f44e',
+    to: '0x3f5CE5FBFe3E9af3971dD833D26BA9b5C936F0bE',
+    data: '',
+    value: '1000000',
+    inputData: {
+        function: 'Mint',
+        contract: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+        parameters: [
+            { type: 'address', value: '0x3f5CE5FBFe3E9af3971dD833D26BA9b5C936F0bE' },
+            { type: 'uint256', value: '2000000000000000000' },
+        ],
+    },
+    receivers: [
+        {
+            currentBalance: 10,
+            newBalance: 20,
+            address: '0x6F46CF5569aefa1acc1009290c8e043747172d89',
+        },
+        {
+            currentBalance: 50,
+            newBalance: 70,
+            address: '0x742d35Cc6634C0532925a3b844Bc454e4438f44e',
+        },
+        {
+            currentBalance: 0,
+            newBalance: 5000,
+            address: '0x53d284357ec70ce289d6d64134dfac8e511c8a3d',
+        },
+    ],
```

Comment:
Generators should only specify minimum required data for the interface
```suggestion
    receivers: [],
```